wkcuber/downsampling.py

```diff
@@ -423,18 +394,17 @@ def downsample_mag(
 def downsample_mags(
     path,
     layer_name,
-    from_mag,
-    max_mag,
+    from_mag: Mag,
+    max_mag: Mag,
     dtype,
     interpolation_mode,
     cube_edge_len,
     jobs,
     compress,
 ):
-    assert log2(from_mag) % 1 == 0, "'from_mag' needs to be power of 2."
-    target_mag = from_mag * 2
+    target_mag = from_mag.scaled_by(2)
```

Comment:
We might want to implement something like "Go from mag 1 to mag 16-16-4", and the code would figure out that the mags are `2-2-1`, `4-4-1`, `8-8-2`, `16-16-4`.

Reply:
Yeah, I thought about this too. I don't know how much this feature is needed, but would be cleaner anyway.